owl/SOMA-OBJ.owl

```diff
@@ -3389,11 +3610,20 @@ Aesthetics is the philosophical study of beauty and taste. The term stems from t
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#TimeRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#TimeRole">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#SpatioTemporalRole"/>
+        <rdfs:comment>A role classifying an entity that specifies the location in time and/or duration of an event or object.</rdfs:comment>
```

Comment:
it sounds a bit like this role may be used to classify time intervals, but that would not be consistent as `Time Interval` is a region